qldbdriver/transaction.go

```diff
@@ -107,20 +107,26 @@ type transactionExecutor struct {
 }
 
 // Execute a statement with any parameters within this transaction.
-func (executor *transactionExecutor) Execute(statement string, parameters ...interface{}) (*Result, error) {
+func (executor *transactionExecutor) Execute(statement string, parameters ...interface{}) (Result, error) {
 	return executor.txn.execute(executor.ctx, statement, parameters...)
 }
 
-// Buffer a Result into a BufferedResult to use outside the context of this transaction.
-func (executor *transactionExecutor) BufferResult(result *Result) (*BufferedResult, error) {
+// Buffer a result into a bufferedResult to use outside the context of this transaction.
+func (executor *transactionExecutor) BufferResult(res Result) (*bufferedResult, error) {
```

Comment:
I think you forgit to add the BufferedResult interface.

We're returning a type we don't export here.

Reply:
Added BufferedResult interface.